docs/specs/toc.yml

```diff
@@ -959,3 +959,23 @@ outputs:
   build.log: |
     ["info","null-value","(Line: 3, Character: 3) 'tocHref' contains null value","docs/TOC.yml",3,3]
 ---
+# Toc with schema vliolation
+inputs:
+  docfx.yml:
+  docs/TOC.yml: |
+    - topicHref: f1/a.md
+      tocHref: 
+      href: f1/b.md
+      additionalProperty: a
+      items: []
+  docs/f1/a.md:
+  docs/f1/b.md:
+outputs:
+  docs/f1/a.json:
+  docs/f1/b.json:
+  build.manifest:
+  build.log: |
+    ["info","null-value","(Line: 2, Character: 3) 'tocHref' contains null value","docs/TOC.yml",2,3]
+    ["error","violate-schema","(Line: 1, Character: 3) Required property 'Name' not found in JSON","docs/TOC.yml",1,3]
+    ["error","violate-schema","(Line: 5, Character: 10) The field JArray must be a string or array type with a minimum length of '1'.","docs/TOC.yml",5,10]
```

Comment:
>The field JArray must be a string or array type with a minimum length of '1' [](start = 56, length = 76)

this error message is not so readable, shall we refine it to:
`Array {propertyName} should contain at least 1 element.`

Reply:
This message comes from data annotation library which is used by a wide range of c# model validation scenarios. I'm happy with the message template, we should just change `JArray` to `{propertyName}`